### ISSUE TITLE
[Suggestion]: bowels are more vertical

### ISSUE THREAD

**Issue by BluePineappleGuy** [nonamecrackers2/crackers-wither-storm-mod#1503]
### Description of feature request

a lot of people say the bowels are far bigger than the mass, so to give it an illusion of actually fitting, i think it should be more vertical like the mass.

**Comment by TaintedBlockmObile** [comment]
When you go in the bowels you go in sideways y'know

**Comment by BluePineappleGuy** [comment]
> When you go in the bowels you go in sideways y'know

do you? the hole in it should send you out the back somewhere, not on the bottom.

**Comment by TaintedBlockmObile** [comment]
> > When you go in the bowels you go in sideways y'know
> 
> do you? the hole in it should send you out the back somewhere, not on the bottom.

It is indeed altered gravity. That's how you even fall out the bottom of the ws

**Comment by BluePineappleGuy** [comment]
> It is indeed altered gravity. That's how you even fall out the bottom of the ws

yes, but the hole points down from the perspective of the player. if you fell in the hole you should fall out somewhere on the back.

**Comment by TaintedBlockmObile** [comment]
> > It is indeed altered gravity. That's how you even fall out the bottom of the ws
> 
> yes, but the hole points down from the perspective of the player. if you fell in the hole you should fall out somewhere on the back.

If you go into the bowels hole from outside, you fall sideways. At this point you are now facing downward since the command block is just farther up the Ws. If the gravity did not change you would go in and land sideways on the floor

**Comment by myron-obsolete-account** [comment]
FYI the storm in MCSM looks so damn small but the bowels are bigger than double the length of the Titanic and I think the bowels are okay as they are, or the WS needs a bigger phase 7 but cracker will probably not do that

**Comment by TaintedBlockmObile** [comment]
I'm aware. And there is no actual way to scale the bowels correctly to the wither storms mass. And even if we could, the model would probably be just too big

**Comment by BluePineappleGuy** [comment]
still would make it more interesting to explore, winding tunnels all over the place. i'm not suggesting that it should fit the model perfectly, i'm suggesting that it's spread in all directions.
